tests/fv/test_multi_host.py

```diff
@@ -0,0 +1,62 @@
+from time import sleep
+import sh
+from sh import docker
+
+
+def multi_host():
+    docker_exec = docker.bake("exec")
+
+    host1_ip = docker.inspect("--format", "'{{ .NetworkSettings.IPAddress }}'", "host1").stdout.rstrip()
+    host2_ip = docker.inspect("--format", "'{{ .NetworkSettings.IPAddress }}'", "host2").stdout.rstrip()
+
+    calicoctl = "/code/dist/calicoctl"
```

Comment:
Some commenting in here would be nice.  Not exactly sure why the "listen" suffix is as it is?  Are we passing in stdin or something?